### ISSUE TITLE
Multiple conditions via text template & formula in 'WHERE' clause

### ISSUE THREAD

**Issue by jegernatt** [alexed1/LightningFlowComponents#266]
I have a use case where I am trying to make a flow re-usable to add Opportunity team members. I'm using the lookupFSC component and I have text template with an embedded formula to appropriately filter the users with a custom attribute set on the User record.

The "Where" clause of the FSC works great when it is manually set, but it breaks when I set the WHERE to a text template (so that a formula can dynamically choose the correct custom attribute to be TRUE and also filter on active users).  The break on the text template happens even if it is set to a 

I can see (in the display (form) and display (tt) in the attached screenshots) that the formula is correctly working and that the text template is properly setting. 

I'd like to achieve this without any apex required if possible, seems like it should work!

Has anyone successfully been able to execute multiple conditions through a formula/text template in a WHERE clause for this component?

Thanks!

![lookupFSC_ttwithformula](https://user-images.githubusercontent.com/60357987/73385314-f588c700-429a-11ea-83b9-93519669c88e.PNG)
![lookupFSC_manuallysetWHERE](https://user-images.githubusercontent.com/60357987/73385315-f7528a80-429a-11ea-8701-504f255c0b5e.PNG)

**Comment by ericrsmith35** [comment]
Be careful with Text Templates.  By default, when you edit a Text Template, it puts you in Rich Text mode.  Be sure to switch to regular text mode and remove any HTML tags that get inserted into the text template.

**Comment by jegernatt** [comment]
Yes — I have been sure to check that the text template is saved as plain instead of rich text. Thanks for the reminder, though.

**Comment by ericrsmith35** [comment]
Here are a few more things to try:

   - Change TRUE to true
   - Put () around each expression
   - Assign the Text Template to a Text variable then use the Text variable
   as your attribute value

**Comment by cchrisv** [comment]
Have you turned on the debug log to see what is actually being inserted into the APEX class?
I would ask mention that formulas and text templates are set before and after the screen, not during. So if you are expecting the value to change based on what is happening on the screen it will not work.

**Comment by adwhite75** [comment]
Speaking of text templates @ericrsmith35, @alexed1 Are you aware of the bug that always flips text templates from Plain Text to Rich Text whenever you make changes to it? Flow doesnt seem to remember your previous preference.

**Comment by ericrsmith35** [comment]
Yes, I have been "bit" by that one far to often.

**Comment by adwhite75** [comment]
@jegernatt were you ever able to grab the apex log to see what is getting passed into apex? Did you check out cchrisv's comment?
